README.md

```diff
@@ -0,0 +1,14 @@
+# **enterprise-frontend-deploy-action**
+
+Action be able to deploy artifacts to an assets but bucket for consumption
```

Comment:
**to** be able?

**but** bucket?

Reply:
That line is copy pasted from the 'About'

Reply:
Ahh! 👍  I've updated the About line.